src/components/video-editor/VideoPlayback.tsx

```
@@ -93,1052 +146,1180 @@ export interface VideoPlaybackRef {
+    ) => {
+      if (!isDraggingFocusRef.current) return;
+      event.preventDefault();
+      updateFocusFromClientPoint(event.clientX, event.clientY);
+    };
+
+    const endFocusDrag = (event: React.PointerEvent<HTMLDivElement>) => {
+      if (!isDraggingFocusRef.current) return;
+      isDraggingFocusRef.current = false;
+      try {
+        event.currentTarget.releasePointerCapture(event.pointerId);
+      } catch {}
```

Biome (2.4.6)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)